hack/util.sh

```diff
@@ -54,6 +46,22 @@ function setup_env_vars {
 		export CURL_CERT="${MASTER_CONFIG_DIR}/admin.crt"
 		export CURL_KEY="${MASTER_CONFIG_DIR}/admin.key"
 	fi
+}
+
+# setup_temp_dir_vars sets ups all temporary directory variables given the subdirectory name
+function setup_temp_dir_vars() {
```

Comment:
I think that this is overkill for some cases.  For instance, I want `test-integration.sh` and `test-go.sh` to log to use a reasonable directory under `TMPDIR`, but I don't see a lot of value in setting the rest of these here.  This seems to cover about 60% of our use, which isn't high enough to warrant it in my mind.

Also, I'm not a fan of `TEMP_DIR`.  It's too close to `TMPDIR` and that has a different meaning.